wordpress-mobile/WordPress-iOS
# New Media Picker: keyboard height fix

This PR addresses the new inline media picker's height not matching the default keyboard's height when predictive text (Quick Type) is enabled. 

Note that [Apple claims](https://developer.apple.com/library/content/documentation/StringsTextFonts/Conceptual/TextAndWebiPhoneOS/InputViews/InputViews.html#//apple_ref/doc/uid/TP40009542-CH12-SW1) setting the `autoresizingMask` to `UIViewAutoresizingFlexibleHeight` on a input view lets iOS automagically set the height of the custom view to match the iOS keyboard, however in practice, this does not work. So this PR tracks the keyboard frame from the standard hide/show notifs.

Fixes #7710 

![test](https://user-images.githubusercontent.com/154014/29668026-4ee67f76-88a4-11e7-8ba6-56f19b4b0c7e.gif)

**To test:**

_Try to test on both the sim and actual devices._

iPhone: 
- Verify the picker matches the same height of the standard keyboard regardless if Predictive Text is turned on or off.
- Try the same thing ^ in landscape/portrait.
- Try the same thing ^ using various international keyboards as well.

iPad: 
- Verify the picker matches the same height of the standard keyboard regardless if Shortcuts are turned on or off.
- Try the same thing ^ in landscape/portrait.
- Try the same thing ^ using various international keyboards as well.
- Make sure the picker still works when a hardware keyboard is attached.

Needs review: @SergioEstevao would you be so kind?
